src/highlight/highlight.interface.ts

```
@@ -0,0 +1,13 @@
+export interface HighlightKey {
+    project: string;
+    timestamp: Number;
```

Biome

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)